content/_help/specific-agencies/internal-revenue-service._en.md

```diff
@@ -0,0 +1,29 @@
+---
+title: IRS (Internal Revenue Service)
+category: specific-agencies
+permalink: /help/specific-agencies/internal-revenue-service/
+order: 3
+---
+
+Your Login.gov account grants you access to IRS web applications that use our service.
+
+Login.gov does not have any information about your taxes or other IRS services. Our program can assist with questions related to creating a Login.gov account, signing in, multi-factor authentication or passwords. Please contact the IRS directly if you have questions regarding:
+
+* Refund status
+* Tax records and tax related information or questions
+* Employer ID Numbers (EIN)
+* Tax payments
+* Tax forms
+
+IRS telephone assistance is available Monday through Friday. Residents of Alaska and Hawaii should follow Pacific time. Phone lines in Puerto Rico are open from 8 a.m. to 8 p.m. local time. Please [visit the IRS telephone assistance website](https://www.irs.gov/help/telephone-assistance) for more information.
```

Comment:
This is more of a grammar question, should "local time" be replaced with Atlantic Standard Time for Puerto Rico?

Reply:
I'm not really sure since I'm implementing the content as provided, but I expect this had undergone a few iterations between our content and communications team, as well as input from the partner.

Reply:
I think local time is fine here because Puerto Rico is in a single zone.